src/others/helpers/decode-aid-request.ts

```diff
@@ -1,15 +1,28 @@
 import omit from "lodash/omit";
 
-import { GroupedByLocationWithTotal, GroupedByCategoryWithTotal } from "./assign-total";
-import { Location, Supply, ID } from "../contexts/api";
+import { Location, Supply, ID, AidRequest } from "../contexts/api";
+
+// TODO: Delete this file once map-utils start using raw AidRequest only
+
+export type GroupedByLocationWithTotal = {
+  total: number;
+  city_id: ID;
+  aidRequests: Omit<AidRequest, "city_id">[];
+};
+
+export type GroupedByCategoryWithTotal = {
+  total: number;
+  category_id: ID;
+  aidRequests: Omit<AidRequest, "category_id">[];
+};
 
 export type DecodedAidRequestGroupedByLocation = {
   location: DecodedLocation;
   total: number;
   decodedAidRequests: DecodedAidRequest[];
 };
 
-export type DecodedLocation = Pick<Location, "name" | "lat" | "lon">;
+export type DecodedLocation = Location;
```

Comment:
why do we need another alias for Location?

Reply:
I think because the map is still using DecodedLocation and before migrating map-utils to new logic we have to get this reference.

Reply:
Yes, it's exactly as @jciepka says. I didn't want this refactor to spill out onto the map.